lib/index.js

```diff
@@ -58,7 +58,8 @@ var epilogue = {
       pagination: args.pagination,
       updateMethod: this.updateMethod,
       search: args.search,
-      sort: args.sort
+      sort: args.sort,
+      reloadOnCreate: args.reloadOnCreate
```

Comment:
It's not necessary for update because on update, the reload is needed for return the good associated object if it is changed.